template/services/{% if "rabbitmq" in athena_services %}{{beamline}}-rabbitmq{% endif %}/values.yaml.jinja

```
@@ -0,0 +1,30 @@
+rabbitmq:
+  auth:
+    username: {{ beamline }}
+    existingPasswordSecret: rabbitmq-secrets
+    existingErlangSecret: rabbitmq-secrets
+  extraPlugins: rabbitmq_jms_topic_exchange,rabbitmq_stomp
+  persistence:
+    enabled: false
+  initScriptsCM: rabbitmq-init
+  rbac:
+    create: false
+  serviceAccount:
+    create: false
+    name: default-full-access-mounted # TODO: use only required permissions
```

```suggestion
    name: default-full-access-mounted
```

Let's make this TODO an issue in this repo to investigate